iphone/Classes/TiUIWebView.m

```diff
@@ -715,8 +715,8 @@ -(CGFloat)contentHeightForWidth:(CGFloat)value
     CGRect oldBounds = [[self webview] bounds];
     BOOL oldVal = webview.scalesPageToFit;
     [webview setScalesPageToFit:NO];
-    [webview setBounds:CGRectMake(0, 0, 10, 1)];
-    CGFloat ret = [webview sizeThatFits:CGSizeMake(10, 1)].height;
+    [webview setBounds:CGRectMake(0, 0, value, 1)];
```

Comment:
This problem is similar to TIMOB-14993. In comment of TIMOB-14993 it is mentioned that with proper mime type, it will work fine else not. But this PR works fine with proper mime type and without mime type (plain text). So I think we should go ahead with this PR.